### PR TITLE
Light drifter button ai

### PR DESCRIPTION
**Pull request by AkashCherukuri**

### CODE REVIEW

File: game/core.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def add_player(self, alias):
 
     def add_bot(self):
         if len(self.players) < 6:
+            self.num_bots+=1
```
Comment (cheekujodhpur) [inline game/core.py]: Why do you need this self.num_bots variable? You can create a function which counts all players with AI flag True

File: game/core.py [hunk 2]
```diff
@@ -311,6 +314,10 @@ def xmlrpc_query_state(self, request, game_id, player_token):
             result["whose_turn"] = game.turn.alias
             result["hand"] = player.hand
             result["top_card"], result["top_card_v"] = game.deck.top_card()
+
+            if(game.turn.token[0].islower()):
```
Comment (cheekujodhpur) [inline game/core.py]: more pythonic syntax please. `if game.turn.token[0].islower():`

Comment (AkashCherukuri) [reply game/core.py]: Did that, will be more "pythonic" from next time

File: game/core.py (hunk 1 above)
Comment (AkashCherukuri) [reply game/core.py]: Okay, yeah I've gone ahead and done that

**Pull request merged**